Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py

```diff
@@ -1048,8 +1049,22 @@ def get_new_indicators(
         new_indicators_list = list(
             filter(last_updated_filter(start_date), new_indicators_list)  # type: ignore
         )
+        return new_indicators_list
+    else:
+        updated_indicators = []
+        # For Indicators of Compromise only
+        for indicator in new_indicators_list:
+            # Check if indicator should be added no matter what
+            # E.g. it meets the `param` requirements
+            if indicator.get("mscore") >= minimum_mscore:
```

Comment:
Do we expect `mscore` is optional here?
Because if it should always be there, let's use `indicator
```suggestion
            if indicator["mscore"] >= minimum_mscore:
```

If it's an optional key, please handle such cases where it's missing.

At its current state, if this key is missing, the condition would become:
`if None >= minimum_mscore:`, which would raise an unexpected error.